frontend/ui/data/koptoptions.lua

```diff
@@ -314,9 +444,6 @@ This can also be used to remove some gray background or to convert a grayscale o
                 toggle = {S.LTR, S.RTL, S.TBRTL},
                 values = {0, 1, 2},
                 default_value = 0,
-                enabled_func = function(configurable)
-                    return optionsutil.enableIfEquals(configurable, "text_wrap", 1)
-                end,
```

Comment:
Now that we have an independant/different Zoom direction toggle for panning, should this one (used to be used with reflow only) still have to be always enabled and no more dependant on reflow/text_wrap ?
(If that's right, please have a quick check any things you did early related to text_wrap are still needed after this switch to an independant toggle for pan direction.)

Reply:
I didn't do anything (as far as I know…) related to text_wrap. The only thing an user will notice is that, if he enables text_wrap after using RT…, the switch will be on `RTL` (or `TBRTL` if applicable).

Reply:
OK, I wrote that when I saw the first chunk of this PR diff, in readercropping.lua : 
```diff
    elseif mode == "none" then
-        if self.document.configurable.text_wrap ~= 1 then
-            self.ui:handleEvent(Event:new("SetZoomMode", "pagewidth", "cropping"))
-        end
```